types/custom-functions-runtime/index.d.ts

```diff
@@ -9,18 +9,17 @@ office-js
 Copyright (c) Microsoft Corporation
 */
 
-/**
- * Specific to Excel Custom Functions.
- * Enables you to set key-value pairs which map
- * a function's id in the JSON metadata to the JS function name.
- * @beta
- */
-declare let CustomFunctionMappings: { [key: string]: Function };
 /**
  * CustomFunctions namespace, used by Excel Custom Functions
  * @beta
  */
 declare namespace CustomFunctions {
+    /**
+     * @beta
+     * Ties together the function's JavaScript name with the JSON id property
+     */
+    function Associate(key: any, functionName: Function): void;
```

Comment:
For the "key", we use "id" in the metadata.json. So we should use "id" here.
Also, the second parameter name should be "function" instead of "functionName"
```suggestion
    function associate(id: string, function: Function): void;
```